tests/test_main.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from main import inner_main
+
+def test_the_program_runs():
+    inner_main(
+        ref_file_path='./tests/data/input/optimization1.out.json',
+        comp_file_path='./tests/data/input/optimization2.out.json',
+        output_path='./tests/.out/',
+        repository_url='https://github.com/chocoteam/choco-solver',
+    )
+
```

Comment:
I'd add assertions on file existence and on file size > 0

Reply:
[Done](https://github.com/chocoteam/PerformanceAction/pull/34/commits/61c3ebf7c1c86693c42ff639942c725548d1f4d2)